diff -ur gnome-phone-manager-0.7-o/src/e-contact-entry.c gnome-phone-manager-0.7/src/e-contact-entry.c
--- gnome-phone-manager-0.7-o/src/e-contact-entry.c	2005-09-09 03:00:18.000000000 -0600
+++ gnome-phone-manager-0.7/src/e-contact-entry.c	2006-08-21 20:08:40.000000000 -0600
@@ -255,9 +255,11 @@
 
       loader = gdk_pixbuf_loader_new ();
 
-      if (gdk_pixbuf_loader_write (loader, (guchar *)photo->data,
-			      photo->length, NULL))
-        pixbuf = gdk_pixbuf_loader_get_pixbuf (loader);
+      //if (gdk_pixbuf_loader_write (loader, (guchar *)photo->data,
+			//    photo->length, NULL))
+      //  pixbuf = gdk_pixbuf_loader_get_pixbuf (loader);
+
+			 pixbuf = NULL;
 
       if (pixbuf) {
           GdkPixbuf *tmp;
